### PR TITLE
feat: upgrade to Palm

### PR DESCRIPTION
**Pull request by Faraz32123**
v16.0.0

### CODE REVIEW

File: tutorcredentials/templates/credentials/build/credentials/Dockerfile
Comment (regisb) [inline tutorcredentials/templates/credentials/build/credentials/Dockerfile:41]: nit: `"{{ OPENEDX_COMMON_VERSION }}"`

File: README.rst
Comment (regisb) [inline README.rst:12]: Can you please replace all instances of "contrib-credentials" by "credentials"? (including in setup.py)

**Review by regisb — CHANGES_REQUESTED** [review]
Thanks for the PR! We'll need to upgrade the discovery plugin to deploy this on the Palm demo instance.

File: .gitlab-ci.yml
Comment (regisb) [inline .gitlab-ci.yml:6]: Please add:

```
TUTOR_EXTRA_ENABLED_PLUGINS: discovery mfe
```

This is required to enable these extra plugins in CI.

File: tutorcredentials/templates/credentials/build/credentials/Dockerfile
Comment (regisb) [inline tutorcredentials/templates/credentials/build/credentials/Dockerfile:136]: Since Dockerize is not used at all anywhere in the plugin, I suggest that you remove all code related to Dockerize.

**Review by regisb — CHANGES_REQUESTED** [review]
Looking good so far! @lpm0073 what do you think?

**Comment by lpm0073** [comment]
This looks GREAT! Thanks for including me in the review process. Looking forward to seeing this merged.

**Comment by lpm0073** [comment]
btw, i'm being presented with a "Merge pull request" action button, which i'm guess might not have been your intention. now that it's been formally adopted by overhangio @regisb would be the sole approver for merge requests, correct?

**Comment by regisb** [comment]
Well Tutor has a maintainer program, where trusted committers can approve and push changes to the plugins: https://discuss.openedx.org/t/tutor-maintainers/7287 You certainly fall in that category :) Would you like to remain a maintainer or would you rather have a "hands-off" approach?

**Comment by lpm0073** [comment]
Good to know, and thank you again for including me. I'll be pleased to remain a maintainer on this as well as any future repos of mine that overhangio might adopt. 

For the merge I'll defer to your better judgement on this particular PR as i haven't sufficiently inspected the syntax and semantics of the code changes, though i'm happy to do that going forward :)

File: README.rst
Comment (Faraz32123) [reply README.rst]: I have changed it a little bit again, @regisb @lpm0073 let me know if its OK.
I have added a Note in it for expanding purpose. Should I keep it or remove it.

**Pull request merged**
